src/sdg/conf/base.py

```diff
@@ -570,3 +561,21 @@
 ZGW_CONSUMERS_CLIENT_CLASS = "sdg.services.client.SDGClient"
 
 SDG_CMS_ENABLED = config("SDG_CMS_ENABLED", default=True)
+
+
+SDG_API_SERVER_INSTANCES = [
+    {
+        "url": "https://pdc.data.vng.nl/sdg",
+        "description": "SDG Testvoorziening",
+    },
+    {
+        "url": "https://pdc-acc.data.vng.nl/sdg",
+        "description": "SDG Acceptatie",
+    },
+    {
+        "url": "https://sdg-invoervoorziening.maykin.nl",
+        "description": "SDG Testvoorziening",
+    },
+]
```

Comment:
Kunnen we dit dynamischer maken door ze als environment var op te geven, mag ongeveer zo:

```
SDG_API_URL_TEST = config("SDG_API_URL_TEST")
SDG_API_URL_ACC = config("SDG_API_URL_ACC")
SDG_API_URL_PROD = config("SDG_API_URL_PROD")

if SDG_API_URL_TEST:
  SDG_API_SERVER_INSTANCES.append(
    {
        "url": SDG_API_URL_TEST,
        "description": "Test omgeving",
    }
  )

etc.